example/Makefile

```diff
@@ -1,30 +1,49 @@
+# It will be == kb / new layout less the targets to work with GO
```

Comment:
@joelanford here I just updated with the kb one and did the change that I think which are required (remove all that is related to gen/check/test/update go code)